src/components/utils/Menu/Menu.tsx

```diff
@@ -9,16 +9,39 @@ interface MenuProps {
   maxHeight: number;
 }
 
+function getOffset(el: HTMLElement) {
+  const rect = el.getBoundingClientRect();
+  return {
+    isLeft: rect.left < window.innerWidth / 2,
+    isBottom: rect.top + window.screenY > window.innerHeight / 2,
```

Comment:
It seems like we dont use this variable do we really need it?

Reply:
We don't I started experimenting with rendering the combobox above but it took way more time than expected.

Removed it.